src/rtrc.js

```diff
@@ -623,6 +627,82 @@
 		$('#krRTRC_loader').hide();
 	}
 
+	function applyOresAnnotations($feedContent, callback) {
+		var revids;
+
+		// Find all revids names inside the feed
+		revids = [];
+		$feedContent.filter('.mw-rtrc-item').each(function () {
+			var revid = $(this).attr('data-diff');
+			if (revid) {
+				revids.push(revid);
+			}
+		});
+
+		if (!revids.length) {
+			callback();
+			return;
+		}
+
+		$.ajax({
+			url: oresApiUrl,
+			data: {
+				// TODO: Update this to new models once they are available
+				models: 'reverted',
+				revids: revids.join('|')
+			},
+			// Todo: reduce this once ores has better performance
+			// https://meta.wikimedia.org/wiki/Research_talk:Revision_scoring_as_a_service#ORES_performance_improvements
+			timeout: 30000,
+			dataType: 'jsonp',
```

Comment:
Using JSONP means cache will still be broken because of the callback parameter. I've filed an issue upstream (https://github.com/wiki-ai/ores/issues/45) to support CORS/JSON instead.